docs/ssh-keys/create-ssh-key-pair.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Create an SSH key pair
+intro: Sometimes you may need to connect to your servers on a new device. Create SSH keys to authenticate your server access on the new device.
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/ssh-keys
+    next:
+        - /docs/ssh-keys/add-ssh-key
+    guides:
+    related:
+        - /docs/ssh-keys/add-ssh-key
+        - /docs/ssh-keys/remove-ssh-key
+    featured:
+---
+
+1. In a terminal window, run command below, substituting in your Devopness email address
+    ```bash
+    ssh-keygen -t rsa -b 4096 -C "<your devopness email address>"
```

Comment:
The e-mail address used as the comment of an SSH key does not need to be the same e-mail associated with the user's account on Devopness, neither needs to be an `@devopness` e-mail address, therefore saying `your devopness email address` might end up being a bit misleading:

```suggestion
    ssh-keygen -t rsa -b 4096 -C "<your email address>"
```